packages/devextreme/testing/tests/DevExpress.ui.widgets/loadPanel.tests.js

```diff
@@ -59,37 +103,40 @@ QUnit.module('init', {
         assert.equal($content.find(MESSAGE_SELECTOR).text(), 'Test Loading Message');
     });
 
-    QUnit.test('role on wrapper', function(assert) {
-        const instance = $('#loadPanel').dxLoadPanel({ visible: true }).dxLoadPanel('instance');
+    QUnit.test('correct wrapper aria attributes load when showIndicator = false', function(assert) {
+        const label = messageLocalization.format('Loading');
+        const $loadPanel = $('#loadPanel').dxLoadPanel({ visible: true, showIndicator: false }).dxLoadPanel('instance');
 
-        assert.strictEqual(instance.$wrapper().attr('role'), 'alert');
+        assert.strictEqual($loadPanel.$wrapper().attr('role'), 'alert', 'role attribute has value alert ');
+        assert.strictEqual($loadPanel.$wrapper().attr('aria-label'), label, 'aria-label attribute has value Loading...');
     });
 
-    QUnit.test('aria-label on wrapper', function(assert) {
-        const instance = $('#loadPanel').dxLoadPanel({ visible: true }).dxLoadPanel('instance');
+    QUnit.test('wrapper aria attributes must not load when showIndicator = true', function(assert) {
+        const $loadPanel = $('#loadPanel').dxLoadPanel({ visible: true, showIndicator: true });
 
-        assert.strictEqual(instance.$wrapper().attr('aria-label'), undefined);
+        assert.strictEqual($loadPanel.attr('role'), undefined, 'role is not set');
+        assert.strictEqual($loadPanel.attr('aria-label'), undefined, 'aria-label is not set');
+    });
 
-        instance.option({ message: '' });
+    QUnit.test('correct load indicator aria attributes load when showIndicator = true', function(assert) {
+        const $loadPanel = $('#loadPanel').dxLoadPanel({ visible: true, showIndicator: true }).dxLoadPanel('instance');
+        const label = messageLocalization.format('Loading');
+        const indicatorAttributes = Array.from($loadPanel.$content().find(LOADINDICATOR_CLASS).prop('attributes')).map(attr => attr.name);
+        const expectedAttributes = ['role', 'aria-label'];
 
-        assert.strictEqual(instance.$wrapper().attr('aria-label'), 'Loading...');
+        assert.strictEqual($loadPanel.$content().find(LOADINDICATOR_CLASS).attr('role'), 'alert', 'role attribute has value alert');
```

Comment:
Let's extract common parts in variables. For example: 
`loadPanelContent = $loadPanel.$content().find(LOADINDICATOR_CLASS)`

Reply:
[816e9ae](https://github.com/DevExpress/DevExtreme/pull/27080/commits/816e9aeb8f98a1be05fe93b205cdef06da5e0bac)